docs/gettingstarted/index.rst

```diff
@@ -189,14 +191,16 @@ Getting started with Flocker
 
          Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
 
-         .. code-block:: console
+         :version-download:`nodes/Vagrantfile.template`
```

Comment:
To confirm, what you want is to have this generate the Vagrantfile but not have a download link, right?